config/environments/assets.rb

```diff
@@ -0,0 +1 @@
+production.rb
```

Comment:
wow, that works?  I thought you had to copy the contents of production.rb here... TIL :wink: